EDA/LibRAG_EDA.ipynb

```diff
@@ -0,0 +1,2124 @@
+{
```

Comment:
<p>Are these all the possible fields? or does every record have every one of these fields?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/BU-Spark/ml-bpl-rag/pull/2/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='3'/>